File: PythonLibs/neonGlowAI.py
# neonGlowAI.py - Uses ONNX Runtime
import onnxruntime as ort

def create_session(model_path=None):
    sess = ort.InferenceSession(model_path or '')
    print('ONNX Runtime session created:', sess)

if __name__ == '__main__':
    create_session()

File: C++Libs/neonGlowAI.cpp
// neonGlowAI.cpp - Uses ONNX Runtime C++ API
#include <onnxruntime_cxx_api.h>
#include <iostream>

int main() {
    Ort::Env env(ORT_LOGGING_LEVEL_WARNING, "test");
    Ort::SessionOptions session_options;
    Ort::Session session(env, "model.onnx", session_options);
    std::cout << "ONNX Runtime session created successfully" << std::endl;
    return 0;
}
